Kernel/makeall.sh

```diff
@@ -18,8 +30,14 @@ if [ "$(uname -s)" = "OpenBSD" ]; then
 	MAKE=gmake
 fi
 
-$MAKE -C ../ clean && \
+if [ $fast_mode = 1 ]; then
     $MAKE -C ../ && \
-    $MAKE -C ../ test && \
-    $MAKE -C ../ install &&
-    sudo -E PATH="$PATH" ./build-image-qemu.sh
+        $MAKE -C ../ install &&
+        sudo -E PATH="$PATH" ./build-image-qemu.sh
+else
+    $MAKE -C ../ clean && \
+        $MAKE -C ../ && \
```

Comment:
What's up with the indentation here?

Reply:
Same indentation as before just with 1 more tab on everything since now it's in a if block.